src/liblgx/liblgx-common/src/sdl/SdlAudioOutput.cpp

```diff
@@ -11,14 +11,13 @@ namespace sdl {
 
     void SdlAudioOutput::initialiseAudio() {
         SDL_AudioSpec want, have;
-
-        SDL_memset(&want, 0, sizeof(want));
+        SDL_zero(want);
         want.freq = 48000;
         want.format = AUDIO_S16LSB;
         want.channels = 2;
         want.samples = 1024;
         want.callback = nullptr;
-        _audio = SDL_OpenAudioDevice(nullptr, 0, &want, &have, SDL_AUDIO_ALLOW_FORMAT_CHANGE);
+        _audio = SDL_OpenAudioDevice(nullptr, 0, &want, &have, 0); //SDL_AUDIO_ALLOW_FORMAT_CHANGE
```

Comment:
If you get rid of the `// SDL_AUDIO_ALLOW_FORMAT_CHANGE` I'll pull this straight in! Thanks a lot dude.